### PR TITLE
feat: Add CI by github actions

### PR DESCRIPTION
**Pull request by erikzenker**
Github action is described in .github/workflows/elixir.yml

### CODE REVIEW

**Comment by erikzenker** [comment]
Once the PR is merged the merge barrier need to be enabled in the repository branch settings by adding a branch protection rule: `Require status checks to pass before merging -> build`

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      uses: actions/setup-elixir@v1
+      with:
+        elixir-version: 1.9.4 # Define the elixir version [required]
+        otp-version: 22.2 # Define the OTP version [required]
+    - name: Install Dependencies
+      run: mix deps.get
+    - name: Compile Dependencies
+      run: mix deps.compile
+    - name: Run Tests
+      run: mix test
```
Comment (farao) [inline .github/workflows/elixir.yml:26]: Please add:
```
name: Check formatting
run: mix format --check-formatted
```

Comment (farao) [reply .github/workflows/elixir.yml:26]: Is it correct, that if `mix format --check-formatted` fails (return value > 0), it does not allow to merge?

Comment (farao) [reply .github/workflows/elixir.yml:26]: Probably the format check should be the first check?

Comment (erikzenker) [reply .github/workflows/elixir.yml:26]: I will add the command. Should be correct. I will test it on my fork.

Comment (erikzenker) [reply .github/workflows/elixir.yml:28]: I added the lines

**Pull request merged**